tests/portals/github/test_manager.py

```diff
@@ -0,0 +1,154 @@
+# type: ignore
+import json
+import os
+import pytest
+from frictionless import portals, FrictionlessException
+from .conftest import options, options_without_dp
+
+
+# Read
+
+
+@pytest.mark.vcr
+def test_github_parser_read(options_without_dp):
+    user = options_without_dp.pop("user")
+    repo = options_without_dp.pop("repo")
+    control = portals.GithubControl()
+    manager = portals.GithubManager(control)
```

Comment:
In the tests we need to use the Package class instead of the Manager class as per requirements. Manager is an internal class; the framework uses under the hood so the user would write (both should work):

```
package = Package(control=portals.GithubControl(user=user, repo=repo))
package = Package('https://github.com/user/repo")
```